rpc_jobs/rpc_aio.yml

```diff
@@ -30,7 +35,9 @@
                     default_volume_type: ceph
             cinder_service_backup_driver: cinder.backup.drivers.ceph
             tempest_service_available_swift: false
-      - upgrade:
+    action:
+      - deploy
+      - majorupgrade:
```

Comment:
we could have kept ``upgrade`` to be non-intrusive, and use (minor) update, leapfrog for the next ones... But we are changing the job names anyway, so I don't think it really matters.